x-pack/filebeat/module/salesforce/logout-rest/config/logout-rest.yml

```diff
@@ -29,18 +29,21 @@ request.proxy_url: {{ .proxy_url }}
 request.transforms:
   - set:
       target: url.params.q
-      value: "SELECT Id,CreatedDate,LogDate,LogFile FROM EventLogFile WHERE Interval = 'Hourly' AND EventType = 'Logout' AND CreatedDate > [[.cursor.last_published_logout]] ORDER BY CreatedDate ASC NULLS FIRST"
-      default: "SELECT Id,CreatedDate,LogDate,LogFile FROM EventLogFile WHERE Interval = 'Hourly' AND EventType = 'Logout' ORDER BY CreatedDate ASC NULLS FIRST"
-response.split:
-  target: body.records
+      value: "SELECT Id,CreatedDate,LogDate,LogFile FROM EventLogFile WHERE Interval = 'Hourly' AND EventType = 'Logout' AND LogDate > [[.cursor.last_published_logout]] ORDER BY CreatedDate ASC NULLS FIRST"

```

Comment:
The orderby clause must be `LogDate` . 
Please make the similar correction in 

https://github.com/elastic/integrations/blob/main/packages/salesforce/data_stream/logout_rest/agent/stream/httpjson.yml.hbs